Analyse_Birds.qmd

```diff
@@ -107,13 +125,220 @@ required_packages <- c(
   "lattice",    # for multipanel graphics
   "dplyr",      # for data manipulation
   "FactoMineR", # for ordinations
-  "factoextra"
+  "factoextra", 
+  "sf",         # for coordinats manipulation
+  "terra",      # for maps creation
+  "spData", 
+  "tmap", 
+  "leaflet", 
+  "spDataLarge", 
+  "maps", 
+  "gifski"      # for gifs display
 )
 
 # Install and load the packages
 install_if_missing(required_packages)
```

Comment:
please again prefer to adapt website.yml. it is not good practice that a script install package (it could be a security problem or the user may not have the right to install package.

Reply:
Modification done, the installation of every package has to be done with the website.yml. Now, if the package isn't installed, this cell prints a message for the user to warn them about the packages installation.